seed/python-sdk/enum/strenum/snippet-templates.json

```diff
@@ -159,15 +160,16 @@
                                             "less_than": "\"less_than\""
                                         },
                                         "templateInput": {
-                                            "location": "BODY",
-                                            "path": "maybeOperandOrColor"
+                                            "location": "RELATIVE",
+                                            "path": null
```

Comment:
Is this correct?

Reply:
Yep!

Reply:
Added a comment to the body to explain why